### PR TITLE
[OWNERS] Update code owners for important directories under aptos-move

### PR DESCRIPTION
**Pull request by movekevin**
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/3236)

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # **Please** keep this file ordered alphabetically by directory paths!
+/aptos-move/framework/ @davidiw @movekevin @wrwg
+/aptos-move/aptos-gas/ @vgao1996
+/aptos-move/aptos-vm/ @davidiw @wrwg
+/aptos-move/vm-genesis/ @davidiw @movekevin
+/aptos-move/parallel-executor/ @davidiw @gelash
```
Comment (davidiw) [inline CODEOWNERS:6]: zekun

File: CODEOWNERS [hunk 2]
```diff
@@ -1,4 +1,9 @@
 # **Please** keep this file ordered alphabetically by directory paths!
+/aptos-move/framework/ @davidiw @movekevin @wrwg
+/aptos-move/aptos-gas/ @vgao1996
+/aptos-move/aptos-vm/ @davidiw @wrwg
```
Comment (davidiw) [inline CODEOWNERS:4]: zekun

**Review by davidiw — APPROVED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (davidiw) [reply CODEOWNERS:6]: not me

File: CODEOWNERS (hunk 2 above)
Comment (movekevin) [reply CODEOWNERS]: Updated

File: CODEOWNERS (hunk 1 above)
Comment (movekevin) [reply CODEOWNERS]: Updated

**Pull request merged**